ui/plugins/controller.js

```diff
@@ -141,6 +152,46 @@ treeherder.controller('PluginCtrl', [
                           }
                           return result;
                         }, []);
+                        if ($scope.job_ids !== null) {
+                            var signatureList = _.keys($scope.job_ids);
+                            var seriesList = [];
+                            $scope.perf_job_detail = [];
```

Comment:
`perf_job_detail` -> `perfJobDetail`.

In general always use camel case when defining a variable in JavaScript.